ops/receipts/2026-04-13/PROJ-313/backup-volume-move-20260414T025733Z/00_summary.md

```
@@ -0,0 +1,83 @@
+# PROJ-313 backup-volume move
+
+- Decision executed: keep Bluesky runtime, Bluesky active Postgres data, Igor production, and Igor staging on root; move backup storage to `/mnt/host-backups`.
+- Control plane used: authenticated DigitalOcean API via existing `DIGITALOCEAN_API_TOKEN`.
+- New volume: `corgi-vps-backups`, 100 GB, attached to `corgi-vps`, mounted at `/mnt/host-backups`.
+
+### Before
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)